docs/releases/5.2.4.txt

```diff
@@ -16,3 +16,7 @@ Bugfixes
 * Fixed a regression in Django 5.2.3 where ``Value(None, JSONField())`` used in
   a :class:`~django.db.models.expressions.When` condition was incorrectly
   serialized as SQL ``NULL`` instead of JSON ``null`` (:ticket:`36453`).
+
+* Relaxed the system check added in Django 5.2 to allow use of
+  ``GeneratedField``\s within a :class:`~django.db.models.CompositePrimaryKey`
+  (:ticket:36466).
```

Comment:
As I feel we should fix ticket-36472 before this change and that that fix would not be back ported (as wasn't an issue introduced in 5.2), we therefore will not back port this change and can remove the release note :+1: 

```suggestion
```